github-workflow/aks-deploy.yaml

```diff
@@ -114,12 +126,23 @@ jobs:
         cluster-name: ${{ steps.aks-cluster-region2.outputs.name }}
         resource-group: rg-bu0001a0042-04
 
+    # Bake using Kubernetes Kustomize - Region 2
+    - name: Bake manifest - Region 2
```

Comment:
Cool, glad this worked out.  (You can resolve this right away)

Reply:
yeah! it is now working fine. I had to commit some extra changes to get it done like indicate what was the Flux namespace. Otherwise Flux will log some errors during the sync loop.